src/vcpkg/base/strings.cpp

```diff
@@ -10,6 +10,20 @@
 #include <string>
 #include <vector>
 
+namespace
+{
+    struct icase_eq_t
+    {
+        bool operator()(char a, char b)
+        {
+            using vcpkg::Strings::details::tolower_char;
+            return tolower_char{}(a) == tolower_char{}(b);
+        }
+    };
+
+    constexpr icase_eq_t icase_eq;
```

Comment:
```suggestion
    constexpr struct
    {
        bool operator()(char a, char b)
        {
            using vcpkg::Strings::details::tolower_char;
            return tolower_char{}(a) == tolower_char{}(b);
        }
    } icase_eq;
```

not necessary but I think it's nicer to just not give it a name.